### ISSUE TITLE
Widget `core-js` which interfers with our app

### ISSUE THREAD

**Issue by davidwessman** [userback/widget-js#59]
Hello.

We are experiencing some issues with a dependency using `window.Promise` and we have tracked it down to when the Userback-widget assets are included.

Have you changed anything around this recently or is it just some incompatiblity?

<img width="875" alt="Screenshot 2023-10-16 at 10 09 51" src="https://github.com/userback/widget-js/assets/6763624/7dcbc4bd-8802-464b-9d4a-a0b102264b9f">

**Comment by lee-userback** [comment]
Yes, we have recently updated to use Webpack 5 to bundle the widget code as well as using core-js as browser polyfills. Are you able to share more details about your issues?

**Comment by davidwessman** [comment]
We have some other dependencies using Angular which relies on overwriting `window.Promise` and now there seems to be a race-condition to which implementation is set.

**Comment by lee-userback** [comment]
Right now, babel is configured for IE11+ where `Promise` is not yet implemented. So `core-js` adds `window.Promise`. The widget code doesn't actually use the native `Promise`. I will see if we can stop `core-js` from adding it.

**Comment by davidwessman** [comment]
> Right now, babel is configured for IE11+ where `Promise` is not yet implemented. So `core-js` adds `window.Promise`. The widget code doesn't actually use the native `Promise`. I will see if we can stop `core-js` from adding it. 

That would be great 🙏

**Comment by lee-userback** [comment]
<img width="852" alt="Screenshot 2023-10-17 at 21 23 02" src="https://github.com/userback/widget-js/assets/33855493/0e537e05-d434-4fdb-b15f-3bdfe2fd390f">

Doing some debugging right now, after the widget code is loaded in the browser, `window.Promise` is the native code instead of a polyfill added by the widget code. @davidwessman are you able to share how you found it was code added by the widget code?

**Comment by davidwessman** [comment]
I'll try, but will probably be by the end of the week.
